archinstall/lib/user_interaction.py

```diff
@@ -1095,3 +1059,122 @@ def generic_multi_select(p_options :Union[list,dict],
 						sort=sort,
 						multi=True,
 						default=default)
+
+
+class UserList(ListManager):
```

Comment:
I think it'd be good to extract this into it's own file, since user_interactions.py is already massive :)

Reply:
Totally Agree. It's just where to put it ...  Perhaps the best solution would be to convert the current file into a directory. It is one of those ideas waiting for time

Reply:
I've opened a discussion #1010 about the future of user_interaction.py. Let's hope we get a good solution on the long term for it